project4/3700crawler

```diff
@@ -88,7 +116,29 @@ class WebCrawler:
     def login(self):
         """Logs into Fakebook"""
         login_form = self.GET(LOGIN_FORM)
-        print(login_form)
+        match = re.search('[Ss]et-[Cc]ookie:\W*csrftoken=(\w+);', login_form)
+        if (not match):
+            print('Couldn\'t extract CSRF token, aborting')
+            print(login_form)
+            exit(1)
+        self.csrf = match.group(1)
+        re.search('[Ss]et-[Cc]ookie:\W*sessionid=(\w+);', login_form)
```

Comment:
Should this say `match = re.search('[Ss]et-[Cc]ookie:\W*sessionid=(\w+);', login_form)` instead? Otherwise, wouldn't `match` still be set to the csrftoken?

Reply:
... And here I was thinking "Wow, it's funny they just use the CSRF token as the session id, good catch